specification/VRMC_springBone-1.0_draft/schema/VRMC_springBone.spring.schema.json

```
@@ -8,17 +8,12 @@
             "type": "string",
             "description": "Name of the Spring"
         },
-        "setting": {
-            "allOf": [ { "$ref": "glTFid.schema.json" } ],
-            "description": "The index of spring settings"
-        },
-        "springRoot": {
-            "allOf": [ { "$ref": "glTFid.schema.json" } ],
-            "description": "The node index of spring root"
-        },
-        "hitRadius": {
-            "type": "number",
-            "description": "The radius of spring sphere"
+        "joints": {
+            "type": "array",
+            "description": "Joints in this spring. Except for the first element, the ancestor of the element must be the previous element of the array",
```

[MUST] ここの説明がアップデートされていません。たぶんこんな感じでしょうか。

```suggestion
            "description": "Joints of the spring. Except for the first element, a previous joint of the array must be an ancestor of the joint.",
```